lib/bolt/transport/winrm/connection.rb

```diff
@@ -105,67 +100,48 @@ def disconnect
           @logger.debug { "Closed session" }
         end
 
-        def shell_init
-          return nil if @shell_initialized
-          result = execute(Powershell.shell_init)
-          if result.exit_code != 0
-            raise BaseError.new("Could not initialize shell: #{result.stderr.string}", "SHELL_INIT_ERROR")
-          end
-          @shell_initialized = true
-        end
-
         def execute(command)
```

Comment:
This should be moved to powershell.rb

Reply:
This method is responsible for executing a command over WinRM. The Powershell `execute` calls this method.